Monika After Story/game/script-stories.rpy

```diff
@@ -0,0 +1,196 @@
+# Module for Monika story telling
+#
+# Stories will get unlocked one at by session
+# The unlocking logic has been added to script-ch30
+# The topic that triggers the Story menu is monika_short_stories
+# Topic is unlocked at the beginning of the game and is not
+# random
+
+
+# dict of tples containing the stories event data
+default persistent.story_database = dict()
+
+
+# store containing stories-related things
+init -1 python in mas_stories:
+
+    # pane constants
+    STORY_X = 680
+    STORY_Y = 40
+    STORY_W = 450
+    STORY_H = 640
+    STORY_XALIGN = -0.05
+    STORY_AREA = (STORY_X, STORY_Y, STORY_W, STORY_H)
+    STORY_RETURN = "I changed my mind"
+    story_database = dict()
+
+
+# entry point for stories flow
+label mas_stories_start:
+    show monika at t21
+    python:
+        import store.mas_stories as mas_stories
+
+        # build menu list
+        stories_menu_items = [
+            (mas_stories.story_database[k].prompt, k, not seen_event(k), False)
+            for k in mas_stories.story_database
+            if mas_stories.story_database[k].unlocked
+        ]
+
+        # also sort this list
+        stories_menu_items.sort()
+
+        # final quit item
+        final_item = (mas_stories.STORY_RETURN, False, False, False, 20)
+
+    # call scrollable pane
+    call screen mas_gen_scrollable_menu(stories_menu_items, mas_stories.STORY_AREA, mas_stories.STORY_XALIGN, final_item=final_item)
+
+    # return value? then push
+    if _return:
+        $ pushEvent(_return)
+
+    show monika at t11
+    return _return
+
+# Stories start here
+init 5 python:
+    addEvent(Event(persistent.story_database,eventlabel="story_tyrant",
+        prompt="The cat and the cock",unlocked=True),eventdb=store.mas_stories.story_database)
+
+label story_tyrant:
+    m 4b "Alright let's start the story."
+    m 3dfc "Ahem."
+    m 1i "A Cat caught a Cock and thought about reasonable excuses for eating him."
+    m "He accused him of being a nuisance by crowing at night; not letting men sleep."
+    m 1d "The Cock defended his action by saying this was for the benefit of men, as it wakes them for labor."
+    m 1tfb "The Cat replied, “you abound in apologies, but it’s time for breakfast.”"
+    m 1l "At that he made a meal of the Cock."
+    m 4d "The moral of this story is that: 'Tyrants need no excuse'."
+    m 1k "I hope you enjoyed this little story, [player]~"
+    return
+
+init 5 python:
+    addEvent(Event(persistent.story_database,eventlabel="story_despise",
+        prompt="The Fox",unlocked=False),eventdb=store.mas_stories.story_database)
+
+label story_despise:
+    m 3k "Ready for story time?"
+    m 3dfc "Ahem."
+    m 1i "One hot summer’s day a Fox was strolling through an orchard till he came to a bunch of Grapes just ripening on a vine which had been trained over a lofty branch."
+    m 1tfu "“Just the thing to quench my thirst,” quoth he."

```

Comment:
You've got some fancy quotes here. Looks like this will work, but if you want to use regular quotes, you can make string using triple quotes:
```python
"""some sort of string with " ' ' " lots of quotes"""
```

Reply:
For this one I used the ones that the stories already had, I like that they are different from the ones the dialogue already has, since that gives the feeling that she's speaking with "another voice". However I think it may be a good idea to ask writers about this too. I'll make changes according to their feedback.